api/data_refinery_api/tests.py

```diff
@@ -169,26 +171,26 @@ def test_search_and_filter(self):
         experiments = []
         for x in range(1, LOTS):
             ex = Experiment()
-            ex.accession_code = "".join(random.choice(words) for i in range(3)) + str(random.randint(0, 1000))[:64]
-            ex.title =  " ".join(random.choice(words) for i in range(10))
+            ex.accession_code = "".join(random.choice(words)
+                                        for i in range(3)) + str(random.randint(0, 1000))[:64]
+            ex.title = " ".join(random.choice(words) for i in range(10))
             ex.description = " ".join(random.choice(words) for i in range(100))
-            ex.technology = random.choice(["RNA-SEQ", "MICROARRAY"])
```

Comment:
Removed why?

Reply:
Because technology is no longer a property on Experiments.

Reply:
Ah, then it should be removed from the two Experiment objects created below this as well

Reply:
I made a mistake here, this property will actually be on both tables so I will readd this.